### ISSUE TITLE
Serial connection doesn't work when multiple IDE windows are open

### ISSUE THREAD

**Issue by AlbyIanna** [arduino/arduino-ide#613]
When you open Serial Monitor or Serial Plotter in multiple IDE windows, they all use the same port, even though you have a different serial port selected from the Board Selector or Tools > Port menu in each window.

**Comment by per1234** [comment]
Hi @AlbyIanna. Can you confirm that, in regards to Serial Monitor, this is a regression that will only be introduced once https://github.com/arduino/arduino-ide/pull/597 is merged (i.e., it does not occur when using any version of the IDE from the `main` branch at this time)?

**Comment by AlbyIanna** [comment]
@per1234 Yes, I confirm it's a bug introduced in the branch `serial-plotter`. We decided to consider this issue an edge case, so we're not planning to solve this before the release. That absolutely does not mean that it's not an important issue, it's just that we have more urgent stuff to do at the moment.

**Comment by per1234** [comment]
Even though this issue is believed to be a new regression, just to create a linkage to related information, I'll add that something similar was mentioned incidentally in another issue:
https://github.com/arduino/arduino-ide/issues/584#issue-1040438820

> If a second IDE is open on another port, the one that looses the connection reconnects to the second IDE's port.

**Comment by per1234** [comment]
Something that might not be clear from this report is that the Serial Monitors show the merged output from the ports selected in each window. There is a demonstration of that here: https://github.com/arduino/arduino-ide/pull/661#issuecomment-986494324

**Comment by mklemarczyk** [comment]
I have same problem, it takes input only from one device and display it in all windows. Even the different boards are connected.
![image](https://user-images.githubusercontent.com/2727261/158821189-fdfb0a21-43b3-47df-b5a8-1458693fca57.png)

I uploaded the program to scan Bluetooth only for 33 BLE. The program on IoT should print only line "starting Bluetooth® Low Energy module failed!" or "Movement detected".

**Comment by wittend** [comment]
This problem is bedeviling me as well.  On Linux Mint (Ubuntu) I'm trying to sort out two sketches operating as client/server on /dev/ttyACMx (ESP32)  communicating with /dev/USBx (Arduino Atmega4808).
